frontend/index.html

```diff
@@ -2,9 +2,9 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="icon" type="image/png" sizes="32x32" href="src/assets/favicon-32x32.png">
```

Comment:
Use the `public` folder (to reference files from the public folder, just use `/favicon...`)  
Please also fix this for Ville's earlier commit and go ahead and delete the whole `src/assets` folder.

You can also clean up all the other unused icons from the `public` folder.

Reply:
Moved to public folder, removed unused icons.